esp/scm/ws_workunits.ecm

```diff
@@ -1827,6 +1837,7 @@ ESPservice [
     ESPmethod [resp_xsl_default("/esp/xslt/WUCopyLogicalFiles.xslt")] WUCopyLogicalFiles(WUCopyLogicalFilesRequest, WUCopyLogicalFilesResponse);
     ESPmethod WUQueryConfig(WUQueryConfigRequest, WUQueryConfigResponse);
     ESPmethod WUListQueries(WUListQueriesRequest, WUListQueriesResponse);
+    ESPmethod [min_ver("1.59")] WUUpdateQuery(WUUpdateQueryRequest, WUUpdateQueryResponse);
```

Comment:
@wangkx The method name "WUUpdateQueryEntry" might be a little better?  The name WUUpdateQuery sounds almost like it might changed what is published, etc.  Rather than just update the field values in the query entry in DALI.